### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by inspecta**
In this milestone,
- I created a new app called `Blog`
- I initialized the project with Git.
- I created and ran the necessary migration files to build the schema
- I included foreign keys and made sure they had an index

### CODE REVIEW

File: db/migrate/20230120060659_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      # t.references :author, null: false, index: true, foreign_key: {to_table: :users}
+      # t.references :post, null: false, index: true, foreign_key: true
+
```
Comment (DammyShittu) [inline db/migrate/20230120060659_create_comments.rb]: - [ ] Kindly remove commented code.

File: db/migrate/20230120075719_add_foreign_keys.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    #Comments
+    add_reference :comments, :author, index: true, foreign_key:{to_table: :users}
+    add_reference :comments, :post, index: true, foreign_key:true
+
+    #Likes
+    add_reference :likes, :author, index: true, foreign_key:{to_table: :users}
+    add_reference :likes, :post, index: true, foreign_key: true
+    # t.references :author, null: false, index: true, foreign_key: {to_table: :users}
+    # t.references :post,  null: false, index: true, foreign_key: true
+
```
Comment (DammyShittu) [inline db/migrate/20230120075719_add_foreign_keys.rb]: - [ ] Kindly remove commented code.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @inspecta,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230120075719_add_foreign_keys.rb (hunk 2 above)
Comment (inspecta) [reply db/migrate/20230120075719_add_foreign_keys.rb]: Thanks for the feedback @DammyShittu

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @inspecta, 

To highlight 👍🏽 
✔️ No linter errors
✔️ Clean code

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**